### PR TITLE
See mp on stats page

### PR DESCRIPTION
**Pull request by crookedneighbor**
I don't think it was left off intentionally, but was unsure.
## Stats Page

screen shot 2015-01-09 at 9 46 42 am
## Member Modal

screen shot 2015-01-09 at 9 46 56 am

Edit: I reversed the images when uploading

### CODE REVIEW

File: views/shared/profiles/stats.jade [hunk 1]
```diff
@@ -4,6 +4,10 @@ table.table.table-striped
     td
       strong=env.t('health')
       | : {{profile.stats.hp | number:0}} / 50
+  tr(ng-if='profile.stats.lvl >= 10 && !profile.preferences.disableClasses')
```
Comment (crookedneighbor) [inline views/shared/profiles/stats.jade:7]: The header uses: `user.flags.classSelected && !user.preferences.disableClasses` to hide MP before a class is picked, but flags aren't accessible through the member service, so I chose the next best thing. If anyone else has better ideas, voice them!

**Comment by crookedneighbor** [comment]
Anyone opposed to this?

**Comment by deilann** [comment]
Some folks have been opposed to it, thinking that if their party members
knew how much MP they had, they'd be pressured into using it (especially
healers).

However, I have repeatedly pointed out that not having MP visible is to
mages as not having HP available would be to healers and iirc, that has
always smoothed over the argument? But I'm pro, so maybe I'm
mis-remembering.

Either way, I think it's a thing that should definitely happen...

**Comment by lemoness** [comment]
I would like to see it for the sake of consistency.

**Pull request merged**